profile/README.md

```diff
@@ -25,10 +25,10 @@ The core libraries are developed and maintained collectively by the Bitcoin Dev
 | `electrum-client`       | [rust-electrum-client](https://github.com/bitcoindevkit/rust-electrum-client) |                    |
 | `esplora-client`        | [rust-esplora-client](https://github.com/bitcoindevkit/rust-esplora-client)   |                    |
 | `bdk-kyoto`             | [bdk-kyoto](https://github.com/bitcoindevkit/bdk-kyoto)                       | [rustaceanrob]     |
-| `BitcoinDevkit` (Swift) | [bdk-ffi](https://github.com/bitcoindevkit/bdk-ffi)                           | [reez]             |
+| `BitcoinDevKit` (Swift) | [bdk-ffi](https://github.com/bitcoindevkit/bdk-ffi)                           | [reez]             |
 | `bdk-android` (Kotlin)  | [bdk-ffi](https://github.com/bitcoindevkit/bdk-ffi)                           | [thunderbiscuit]   |
 | `bdk-jvm` (Kotlin)      | [bdk-ffi](https://github.com/bitcoindevkit/bdk-ffi)                           | [thunderbiscuit]   |
-| `bdkpython` (Python)    | [bdk-ffi](https://github.com/bitcoindevkit/bdk-ffi)                           | [thunderbiscuit]   |
+| `bdk-python` (Python)   | [bdk-ffi](https://github.com/bitcoindevkit/bdk-ffi)                           | [thunderbiscuit]   |
```

Comment:
In this case the library is really called `bdkpython`! The directory in the bdk-ffi repo is called bdk-python for consistency with the other directories, but the library itself is called bdkpython.